src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/PersonSentencesTest.kt

```diff
@@ -22,6 +24,44 @@ class PersonSentencesTest : DescribeSpec(
         integrationApiSentence.isActive.shouldBe(true)
       }
 
+      it("maps Nomis terms to Integration API terms") {
+        val nomisSentence = NomisSentence(
+          terms = listOf(
+            NomisTerm(
+              years = 3,
+              months = 4,
+              weeks = 0,
+              days = 2,
+            ),
+            NomisTerm(
+              years = 7,
+              months = 3,
+              weeks = 4,
+              days = 0,
+            ),
+          ),
+        )
```

Comment:
You need to have a test case where you test NomisTerms where any combination of those fields are missing.
I.e. years = 3 in one, months = 2 in the other.
You also need a case where you test a completely empty NomisTerm (because it is possible for no terms to exist)
You also need a case where you test a single Term in a list, to ensure smooth functionality for the singular case.